JulioC-19/LargeProjectMobile
# Julio/input validation

Implemented input validation and toast indicators to sign up and login screens, ingredient parsing for recipe screens and favorites. A toast is a timed self-hiding pop up at the top of the screen, different messages and colors appears depending on the status call of the API calls.

To test: 
Go to signup screen and try to fill invalid inputs and press the button, the text-inputs should turn red if they are invalid and 
the API should not be executed. 
The background will remain red until a valid input is typed in for each field. 
If the passwords don't match it will also trigger an invalid input. 
Play around with the different inputs to notice the different error warnings. 

In login screen:
Try to input incorrect username or password, the error toasts will appear. 
Try the correct login, a success toast should appear.
